src/rules/extract-global-constants.ts

```
@@ -24,62 +24,66 @@ function isFunctionDefinition(node: TSESTree.Expression | null): boolean {
+        | undefined;
+      if (!inner) break;
+      current = inner;
+      continue;
+    }
+    break;
   }
+  return current;
 }
 
+
 function isMutableValue(node: TSESTree.Expression | null): boolean {
```

ESLint

[error] Delete `⏎`

(prettier/prettier)